### ISSUE TITLE
D2SCS Scene Controller Buttons always Unknown/Unavailable

### ISSUE THREAD

**Issue by dannyvfilms** [schmittx/home-assistant-leviton-decora-smart-wifi#3]
I was happy to find this integration to hopefully be able to use the [Scene Controller Buttons](https://leviton.com/products/d2scs-1bw) of my D2SCS in Home Assistant instead of through HomeKit. If I manually press the buttons in the Home Assistant interface my automations trigger, but the light switch doesn't seem to be sending any data whenever I press the buttons on the switch itself. 

The main load control switch works properly and updates when pressed on the light switch, so the issue is only with the scene controller buttons. All switches are on the latest 1.6.4 firmware.

**Comment by schmittx** [comment]
Yeah, this is a known limitation as the integration works by intermittently polling the Leviton API. There is a websocket connection available, which the official uses, that pushes updates from the API but that's not implemented in this integration yet.

**Comment by mxfire** [comment]
I have been also trying to trigger an action in HA when I press the buttons on the D2SCS. Any plans get the websocket connection to work ? The official Leviton integration is useless, yours is working so much better. thanks.

**Comment by schmittx** [comment]
Yeah, websocket support is the ultimate goal but I personally have very little experience with that and async operation, so it's a big learning curve. If someone else wants to give it a go, please feel free to.

**Comment by schmittx** [comment]
DONE! Thanks to @InnovoDeveloper! Check out [2.0.0](https://github.com/schmittx/home-assistant-leviton-decora-smart-wifi/releases/tag/2.0.0)
